lib/index.ts

```diff
@@ -348,10 +348,26 @@ const safeReadFile = async (filename: string): Promise<Buffer | undefined> => {
 	}
 };
 
+const unsafeReadFile = async (filename: string): Promise<Buffer | undefined> => {
+	try {
+		return await readFile(filename);
+	} catch (e) {
+		// no data
+	}
+};
+
 const getFileBuffer = async (
 	device: usb.Device,
 	filename: string,
+	extraFolder?: string,
 ): Promise<Buffer | undefined> => {
+	if(extraFolder) {
+		const extraBuffer = await unsafeReadFile(Path.join(extraFolder, filename));
```

Comment:
You don’t seem to really need the unsafeReadFile function, just inlining the readFile call and wrapping in try/catch instead of the if check below is at least as clear. What do I miss?

Reply:
Indeed

We don't really need the "safeReadFile" either, I refactored both out.